### PR TITLE
some 14a stuff

### PR DESCRIPTION
**Pull request by Shnupbups**
Alright, that's all the time I've got. I've got to get back to playing Animal Crossing: New Horizons on my Nintendo Switch.

### CODE REVIEW

File: mappings/net/minecraft/entity/mob/Hoglin.mapping [hunk 1]
```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/class_5137 net/minecraft/entity/mob/Hoglin
```
Comment (liach) [inline mappings/net/minecraft/entity/mob/Hoglin.mapping:1]: What is this? `ZoglinEntity` or `HoglinLogic`?

**Review by liach — APPROVED** [review]

File: mappings/net/minecraft/entity/mob/Hoglin.mapping (hunk 1 above)
Comment (Shnupbups) [reply mappings/net/minecraft/entity/mob/Hoglin.mapping:1]: It's an interface that both `HoglinEntity` and `ZoglinEntity` implement. Couldn't think of a good name lol

File: mappings/net/minecraft/enchantment/EnchantmentHelper.mapping [hunk 2]
```diff
@@ -17,6 +17,8 @@ CLASS net/minecraft/class_1890 net/minecraft/enchantment/EnchantmentHelper
 			COMMENT the entity to choose equipments from
 		ARG 2 condition
 			COMMENT extra conditions for the item stack to pass for selection
+	METHOD method_24366 (Lnet/minecraft/class_1799;)Z
```
Comment (YanisBft) [inline mappings/net/minecraft/enchantment/EnchantmentHelper.mapping:20]: Mind map this method?

Comment (liach) [reply mappings/net/minecraft/enchantment/EnchantmentHelper.mapping:20]: It's the lambda in `chooseEquipment`, so the method cannot be mapped, only its lambda param can be

File: mappings/net/minecraft/entity/attribute/EntityAttributes.mapping [hunk 3]
```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/class_5134 net/minecraft/entity/attribute/EntityAttributes
+	METHOD method_26871 register (Ljava/lang/String;Lnet/minecraft/class_1320;)Lnet/minecraft/class_1320;
+		ARG 0 id
+		ARG 1 entityAttribute
```
Comment (YanisBft) [inline mappings/net/minecraft/entity/attribute/EntityAttributes.mapping]:
```suggestion
		ARG 1 attribute
```

File: mappings/net/minecraft/entity/mob/HoglinEntity.mapping [hunk 4]
```diff
@@ -3,9 +3,24 @@ CLASS net/minecraft/class_4760 net/minecraft/entity/mob/HoglinEntity
 	FIELD field_22366 MEMORY_MODULE_TYPES Lcom/google/common/collect/ImmutableList;
 	FIELD field_22367 LOGGER Lorg/apache/logging/log4j/Logger;
 	FIELD field_22368 movementCooldownTicks I
+	FIELD field_23735 timeInOverworld I
+	FIELD field_23736 cannotBeHunted Z
+	FIELD field_23737 BABY Lnet/minecraft/class_2940;
 	METHOD method_24349 canSpawn (Lnet/minecraft/class_1299;Lnet/minecraft/class_1936;Lnet/minecraft/class_3730;Lnet/minecraft/class_2338;Ljava/util/Random;)Z
+		ARG 0 entityType
```
Comment (YanisBft) [inline mappings/net/minecraft/entity/mob/HoglinEntity.mapping]:
```suggestion
		ARG 0 type
```

File: mappings/net/minecraft/entity/mob/HoglinEntity.mapping [hunk 5]
```diff
@@ -3,9 +3,24 @@ CLASS net/minecraft/class_4760 net/minecraft/entity/mob/HoglinEntity
...
 		ARG 1 world
+		ARG 2 spawnType
 		ARG 3 pos
 	METHOD method_24658 isAdult ()Z
 	METHOD method_24659 playFightSound ()V
 	METHOD method_24661 playRetreatSound ()V
+	METHOD method_26942 zombify (Lnet/minecraft/class_3218;)V
+		ARG 1 word
+	METHOD method_26944 canConvert ()Z
+	METHOD method_26945 getCanBeHunted ()Z
```
Comment (YanisBft) [inline mappings/net/minecraft/entity/mob/HoglinEntity.mapping]: Hum, not a good name for a boolean getter

Comment (Runemoro) [reply mappings/net/minecraft/entity/mob/HoglinEntity.mapping]: Should just be `canBeHunted`

Comment (liach) [reply mappings/net/minecraft/entity/mob/HoglinEntity.mapping]:
```suggestion
	METHOD method_26945 canBeHunted ()Z
```

File: mappings/net/minecraft/entity/mob/HoglinEntity.mapping [hunk 6]
```diff
@@ -3,9 +3,24 @@ CLASS net/minecraft/class_4760 net/minecraft/entity/mob/HoglinEntity
...
+	METHOD method_26946 immuneToZombification ()Z
```
Comment (liach) [inline mappings/net/minecraft/entity/mob/HoglinEntity.mapping]:
```suggestion
	METHOD method_26946 isImmuneToZombification ()Z
```

**Review by liach — COMMENTED** [review]
@Shnupbups just apply the suggested changes from yanis and me and looks good

**Review by liach — APPROVED** [review]

**Review by YanisBft — APPROVED** [review]

**Pull request merged**